<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="95" columnstretch="8,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="packingOfMainWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">#packingOfMainWidget #mainWidget{
	border-style:solid;
	border-width:2;
	border-color:blue;
	border-radius:10;
}

#packingOfMainWidget{
	border-style:solid;
	border-width:2;
	border-color:red;
	border-radius:10;
}
#next_btn{
	border-style:solid;
	border-width:1;
	border-color:lightgrey;
	background-color: rgba(0, 0, 0, 0);
	
	border-top-left-radius:10;
	border-bottom-left-radius:10;
}
#previous_btn{
	border-style:solid;
	border-width:1;
	border-color:lightgrey;
	background-color: rgba(0, 0, 0, 0);
	
	border-top-right-radius:10;
	border-bottom-right-radius:10;
}
#previous_btn:hover{
	background-color:rgba(96,178,255,50);
	color:white
}

#previous_btn:pressed{
	margin-top:2px;
	margin-right:2px;
	background-color:rgba(59,110,158,50);
}

#next_btn:hover{
	background-color:rgba(96,178,255,50);
	color:white
}

#next_btn:pressed{
	margin-top:2px;
	margin-left:2px;
	background-color:rgba(59,110,158,50);
}</string>
      </property>
      <widget class="AnimationStackedWidget" name="mainWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>800</width>
         <height>400</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>400</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page"/>
       <widget class="QWidget" name="page_2"/>
      </widget>
      <widget class="QPushButton" name="previous_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="next_btn">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QDateTimeEdit" name="beginTime_DTE">
           <property name="displayFormat">
            <string>yyyy/M/d H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="endTime_DTE">
           <property name="displayFormat">
            <string>yyyy/M/d H</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="CalendarWidget" name="beginTime_CW" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>266</width>
          <height>255</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>266</width>
          <height>255</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CalendarWidget" name="endTime_CW" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>266</width>
          <height>255</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>266</width>
          <height>255</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AnimationStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>animationstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalendarWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/calendarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
